ext/standard/string.c

```diff
@@ -5632,6 +5632,7 @@ PHP_FUNCTION(substr_count)
 			length += (haystack_len - offset);
 		}
 		if (length < 0 || ((size_t)length > (haystack_len - offset))) {
+			/* TODO Check if Candidate to convert to Exception */
 			php_error_docref(NULL, E_WARNING, "Invalid length value");
```

Comment:
why is this warning not converted? invalid length is at least as serious as empty substring IMO.